### PR TITLE
User guide: PED Bug Fixes

### LINKED ISSUE #194: [PE-D] Add feature : Level
I notice that there is not standard levels tag in place (eg: secondary 1, secondary 2 or sec 1 , sec 2 etc)

Meaning that I can input secondary 1, sec1 and sec 1 or sec one. They are all valid inputs, which perharps might not be the best here !

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: mrweikiat/ped#11

### LINKED ISSUE #200: [PE-D] Current date's session implies that tutor is having a session now but i think it means current date and time?
No details provided.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Winniehyx/ped#6

### LINKED ISSUE #204: [PE-D] Not clear in documentation that add_session comes after add_student
No details provided.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Winniehyx/ped#10

### LINKED ISSUE #213: [PE-D] User guide Standardisation
For user guide, will it be better to standardise to a common name like John Doe. This makes it easier to copy paste and test all the functionalities. This is because some of the command cannot copy paste and test due to differences in the current data stored in the system

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: CSjiade/ped#3

### PR DESCRIPTION
**Pull request by JonahhGohh**
Closes #213, closes #204, closes #200, closes #194

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -234,6 +239,10 @@ Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEV
 <div markdown="block" class="alert alert-info">
 :information_source: <code>STUDENT_PHONE_NUMBER</code>, <code>GUARDIAN_PHONE_NUMBER</code> should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and with 8 digits)
 </div>
+<div markdown="block" class="alert alert-primary">
+:bulb: TutorBuddy does not restrict the <code>STUDY_LEVEL</code> due to the vast number of education levels. The user
```
Comment (enhao25) [inline docs/UserGuide.md:243]: Hi, I was thinking if we could add in some example, "...number of education levels (E.g. Sec 2 / Secondary 2 / Computing / CS1101S)"

**Review by enhao25 — APPROVED** [review]
LGTM! Only have 1 suggestion that perhaps we could add in some examples for the study level portion.

File: docs/UserGuide.md (hunk 1 above)
Comment (JonahhGohh) [reply docs/UserGuide.md:243]: Sure!

**Review by chwoozy — APPROVED** [review]
LGTM! 🍾

**Pull request merged**